spec/rubocop/cop/rspec/receive_messages_spec.rb

```diff
@@ -0,0 +1,54 @@
+# frozen_string_literal: true
+
+RSpec.describe RuboCop::Cop::RSpec::ReceiveMessages, :config do
+  it 'registers an offense when multiple messeages stubbed on the ' \
+     'same object' do
+    expect_offense(<<~RUBY)
+      before do
+        allow(Service).to receive(:foo).and_return(baz)
+        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ Use `receive_messages` instead of multiple stub on line(s) [3, 4].
+        allow(Service).to receive(:bar).and_return(bar)
```

Comment:
What if both those allow to receive the same `foo` message?

Reply:
It would be better to create a separate cop to check for duplicate messages and leave it to them. As for this cop, I will try to ignore duplicate messages. What do you think?

Reply:
There’s an ‘ordered’ qualifier that makes sense with duplicates, along with ‘exactly(count)’ and ‘once’/‘twice’.